cmds/core/wget/wget_test.go

```diff
@@ -98,25 +98,21 @@ var tests = []struct {
 	},
 }
 
-func getFreePort(t *testing.T) int {
+func getFreePort(t *testing.T) (net.Listener, int) {
 	l, err := net.Listen("tcp", ":0")
```

Comment:
nit: Please add t.Helper() call here (https://pkg.go.dev/testing#T.Helper) - this will help with debugging test failures.